### PR TITLE
76-image-profile-uploader: implement image profile uploader

### PR DESCRIPTION
**Pull request by adibNo9**
- update palette
- add profileImage icon
- implement component, logic, ui

### CODE REVIEW

File: libs/components/src/lib/ImageProfile/index.hook.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+export const useData = () => {
+  const inputRef = useRef<HTMLInputElement>(null);
+  const [profileImage, setProfileImage] = useState<string>("");
+
+  const uploadHandler = () => {
+    inputRef.current?.click?.();
+  };
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    e.target.files && setProfileImage(URL.createObjectURL(e.target.files[0]));
```
Comment (amirmohammad-mohamadi) [inline libs/components/src/lib/ImageProfile/index.hook.ts:16]: if (!e.target.files) return;
return setProfileImage(URL.createObjectURL(e.target.files[0]));

File: libs/components/src/lib/ImageProfile/index.module.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
+@import "../../../../theme/index";
+
+.container {
+    position: relative;
+    width: 250px;
+    height: 250px;
+    background-color: var(--grey-50);
+    border-radius: 100%;
+    overflow: hidden;
+    display: flex;
```
Comment (amirmohammad-mohamadi) [inline libs/components/src/lib/ImageProfile/index.module.scss:10]: use mixin flexbox

File: libs/components/src/lib/ImageProfile/index.module.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    cursor: pointer;
+
+    .text__wrapper {
+        h6 {
+            margin: 0;
```
Comment (amirmohammad-mohamadi) [inline libs/components/src/lib/ImageProfile/index.module.scss:18]: use spacing(0)

File: libs/components/src/lib/ImageProfile/index.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+export const ImageProfile = () => {
+  const { inputRef, profileImage, handleChange, uploadHandler } = useData();
+
+  return (
+    <div className={classes.container} onClick={uploadHandler}>
+      <input
+        ref={inputRef}
+        type="file"
+        onChange={handleChange}
+        accept=".png,.jpeg"
```
Comment (amirmohammad-mohamadi) [inline libs/components/src/lib/ImageProfile/index.tsx:16]: define all image types

File: libs/components/src/lib/ImageProfile/index.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      />
+      <ProfileImageIcon />
+      <Typography variant="h6" rootClassName={classes.text__wrapper}>
+        {texts.yourPicture}
+      </Typography>
+      {profileImage && <img src={profileImage} alt="profile-img" />}
```
Comment (amirmohammad-mohamadi) [inline libs/components/src/lib/ImageProfile/index.tsx:22]: use shadow rendering like this:
```
const renderImage = () => {
if (!profileImage) return;
return <img src={profileImage} alt="profile-img" />
}
```

File: libs/components/src/lib/ImageProfile/index.tsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    </div>
+  );
+};
+
+export default ImageProfile;
```
Comment (amirmohammad-mohamadi) [inline libs/components/src/lib/ImageProfile/index.tsx:27]: remove this

File: libs/components/src/lib/Icons/profileImage.tsx [hunk 7]
```diff
@@ -0,0 +1,21 @@
+import { SVGProps } from "react";
+
+export const ProfileImageIcon = (props: SVGProps<SVGSVGElement>) => (
+  <svg
+    {...props}
```
Comment (amirmohammad-mohamadi) [inline libs/components/src/lib/Icons/profileImage.tsx:5]: add {...props} to end of svg tag

File: libs/components/src/lib/ImageProfile/index.module.scss (hunk 2 above)
Comment (adibNo9) [reply libs/components/src/lib/ImageProfile/index.module.scss:10]: done

File: libs/components/src/lib/ImageProfile/index.hook.ts (hunk 1 above)
Comment (adibNo9) [reply libs/components/src/lib/ImageProfile/index.hook.ts:16]: done

File: libs/components/src/lib/ImageProfile/index.module.scss (hunk 3 above)
Comment (adibNo9) [reply libs/components/src/lib/ImageProfile/index.module.scss:18]: done

File: libs/components/src/lib/ImageProfile/index.tsx (hunk 4 above)
Comment (adibNo9) [reply libs/components/src/lib/ImageProfile/index.tsx:16]: done

**Pull request merged**